test/compress/global_defs.js

```diff
@@ -123,6 +126,8 @@ mixed: {
         console.log(CONFIG);
     }
     expect: {
+        const FOO = { BAR: 0 };
+        console.log(FOO.BAR);
```

Comment:
I'm slightly confused now - `FOO.BAR` is defined as `"moo"` In `global_defs`, so do you mean you want that instead of `0`?